### PR TITLE
feat(time): 'Added minutes algorithm to time'

### PR DESCRIPTION
**Pull request by abhishek-mishra019**
Signed-off-by: Abhishek Mishra <am1426620@gmail.com>
### Functionality:
to add minutes in time fields such as Estimated time, Log Time, Remaining Time

### Solution:
implemented the algorithm for hours and minutes in time

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
step 1: try creating a task with hours and minutes time fields, 
step 2: see the time details on task details page.
step 3: try logging the time with hours and minutes time fields
step 4: try edit task with hours and minutes time fields
step 5: try different ways to enter the time as only hours or only minutes or both.

### CODE REVIEW

File: src/app/body/task-details/edit-page/edit-page.component.ts [hunk 1]
```diff
@@ -118,6 +127,29 @@ export class EditPageComponent implements OnInit {
           });
     }); 
   }
+  handleMinsHrs(){
```
Comment (viv9k) [inline src/app/body/task-details/edit-page/edit-page.component.ts:130]: We can optimise this code by moving it into tools Service

**Review by viv9k — CHANGES_REQUESTED** [review]
Please check and let me know

File: src/app/body/task-details/edit-page/edit-page.component.ts (hunk 1 above)
Comment (abhishek-mishra019) [reply src/app/body/task-details/edit-page/edit-page.component.ts]: pushed the changes

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**